plugin/vim-gh-line.vim

```diff
@@ -285,8 +319,11 @@ func! s:CgitUrl(origin)
                 \ 'g:gh_cgit_url_pattern_sub:' . string(g:gh_cgit_url_pattern_sub)
 endfunc
 
-noremap <silent> <Plug>(gh-line) :call <SID>gh_line('blob')<CR>
-noremap <silent> <Plug>(gh-line-blame) :call <SID>gh_line('blame')<CR>
+noremap <silent> <Plug>(gh-line) :call <SID>gh_line('blob', 0)<CR>
+noremap <silent> <Plug>(gh-line-blame) :call <SID>gh_line('blame', 0)<CR>
+
+command! GHIteractive call <SID>gh_line('blob', 1)
+command! GBIteractive call <SID>gh_line('blame', 1)
 
 if !hasmapto('<Plug>(gh-line)') && exists('g:gh_line_map')
     exe "map" g:gh_line_map "<Plug>(gh-line)"
```

Comment:
Could we add gh_line_interactive_map variable too ? 
That will add full configurability to this plugin.

Reply:
we could, but I personally don't want to add too many default key binding to this tiny plugin. If user need, they can still have mapping to `GHIteractive`

Reply:
ok Fair enough.